.github/workflows/ci.yml

```diff
@@ -21,7 +21,7 @@ permissions:
 
 jobs:
   dev-ci:
-    uses: xaynetwork/xayn_discovery_engine/.github/workflows/ci_reusable_wf.yml@main
+    uses: ./.github/workflows/ci_reusable_wf.yml
```

Comment:
Does this really work now? When we started doing this it wasn't possible. Would be very nice!

Reply:
Can you please also do this change in the bors ci?

Reply:
At least for me, on my branch, this has been working fine :)